data/mods/MindOverMatter/powers/combination_powers/pyrokinesis.json

```diff
@@ -0,0 +1,179 @@
+      ]
+    },
+    "max_range": 80,
+    "min_aoe": {
+      "math": [
+        "min( (( (u_spell_level('pyro_photokinetic_combo_flash') * 1.2) + 6) * (scaling_factor(u_val('intelligence') ) ) * u_nether_attunement_power_scaling), 8)"
+      ]
+    },
+    "max_aoe": 30,
+    "min_duration": {
+      "math": [
+        "( (u_spell_level('pyro_photokinetic_combo_flash') * 1000) + 5000) * (scaling_factor(u_val('intelligence') ) ) * u_nether_attunement_power_scaling"
+      ]
+    },
+    "max_duration": {
+      "math": [
+        "( (u_spell_level('pyro_photokinetic_combo_flash') * 1100) + 7000) * (scaling_factor(u_val('intelligence') ) ) * u_nether_attunement_power_scaling"
+      ]
+    },
+    "energy_source": "STAMINA",
+    "base_energy_cost": 2500,
+    "final_energy_cost": 1200,
+    "energy_increment": -109,
+    "base_casting_time": 200,
+    "final_casting_time": 100,
+    "casting_time_increment": -8,
+    "ignored_monster_species": [ "PSI_NULL" ]
+  },
+  {
+    "id": "pyro_photokinetic_combo_fireball",
+    "type": "SPELL",
+    "name": "[Ψ]Sunspot Fireball",
+    "description": "Cause a burst of flames for Sunspot.  It is a bug if you have this.",
+    "valid_targets": [ "hostile", "ground", "ally" ],
+    "flags": [ "PSIONIC", "CONCENTRATE", "LOUD", "NO_PROJECTILE", "IGNITE_FLAMMABLE", "RANDOM_DAMAGE", "NO_HANDS", "NO_LEGS" ],
+    "max_level": { "math": [ "int_to_level(1)" ] },
+    "spell_class": "PYROKINETIC",
+    "effect": "attack",
+    "shape": "blast",
+    "damage_type": "heat",
+    "difficulty": 0,
+    "min_damage": {
+      "math": [
+        "( (u_spell_level('pyro_photokinetic_combo_fireball') * 1.5) + 6) * (scaling_factor(u_val('intelligence') ) ) * u_nether_attunement_power_scaling"
+      ]
+    },
+    "max_damage": {
+      "math": [
+        "( (u_spell_level('pyro_photokinetic_combo_fireball') * 3) + 12) * (scaling_factor(u_val('intelligence') ) ) * u_nether_attunement_power_scaling"
+      ]
+    },
+    "min_range": {
+      "math": [
+        "clamp( ( (u_spell_level('pyro_photokinetic_combo_fireball') + 1) * (scaling_factor(u_val('intelligence') ) ) * u_nether_attunement_power_scaling), 1, 5)"
+      ]
+    },
+    "min_aoe": {
+      "math": [
+        "min( (( (u_spell_level('pyro_photokinetic_combo_flash') * 0.25) + 3) * (scaling_factor(u_val('intelligence') ) ) * u_nether_attunement_power_scaling), 8)"
+      ]
+    },
+    "max_aoe": 10,
+    "max_range": 12,
+    "ignored_monster_species": [ "PSI_NULL" ]
+  },
+  {
+    "id": "pyro_biokinetic_dragon_breath",
+    "type": "SPELL",
+    "//": "Requires Pyrokinesis and Biokinesis to learn.",
+    "name": "[Ψ]Dragon's Spittle",
+    "description": "Your body synthesizes a flammable, sticky goop, which is expelled from one of your orificies and ignited.  Causes a raging inferno which sticks around, slowing down enemies caught in the blast..",
```

Comment:
This is definitely much too much like magic for a psionic power. The power limitations guide says that biokinesis cannot be used to allow the body to do anything completely outside its normal capabilities, which organic napalm definitely fits under.

Reply:
Come to think of it, would it be possible to gate the acquisition/use of this power behind mutations that allow the player to produce webs or other flammable substances, like some of the spider lines? I.e you can't learn it unless you have the web-spinner mutation.
Can psionic powers even interact with mutations in this way, or does that also break lore constraints?

Reply:
_That_ would be a good solution, since there are already psionic animals in the mod. If you can already produce webs, then that's within your normal capabilities and it fits the design docs.

Reply:
Consider it done, then. I'll rig this into any learning EOCs I might make with it.

Reply:
Also, this finally gave me an idea for a psionic spider to make, so thank you!

Reply:
Oh, you're quite welcome!